assets/trix/stylesheets/functions.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path')
+const fs   = require('fs')
+const sass = require("node-sass")
+const { optimize } = require('svgo')
+
+const types = sass.types
+const basePath = "./assets/"
+
+module.exports = {
+  'svg($svgFileName)': function (svgFileName) {
+      const filename = path.resolve(basePath, svgFileName.getValue())
+
+      let svgContent = fs.readFileSync(filename, "utf8")
+      svgContent = optimize(svgContent, { multipass: true, datauri: 'enc' })
+
+      return new types.String(`url("${svgContent.data}")`)
+  }
+}
```

Comment:
This is clever, but I guess we could also just svgo optimize the icons once and for all? Although suppose this would keep a system in place to ensure that it always happens going forward.

Reply:
Oh, I thought you could just use the svg() method you get from node-sass-svg? Either way, I think this is totally fine, and the optimization step is a nice guarantee that even if we optimize icons now, we don't forget something in the future. It's not a runtime issue anyway.

Reply:
Oh, I couldn't make node-sass-svg work with these versions of node-sass/node. 

Removed in  dea4e28439e4979118f9db931562e54f7fab6815